src/tools.py

```diff
@@ -727,6 +727,27 @@ def tools_basic_space_cleanup() -> None:
         shell=True,
     )
 
+def tools_clean_old_kernels() -> None:
+    # Removing kernel except last one
+    uname_output = subprocess.run("uname -r", capture_output=True, text=True, shell=True)
+    # running_kernel is expected to be something like : 6.1.0-28-amd64
+    running_kernel = uname_output.stdout.strip()
+    dpkg_output = subprocess.run("dpkg -l | grep '^ii\s*linux-image-[0-9]' | awk '{print $2}'", capture_output=True, text=True, shell=True)
```

Comment:
```suggestion
    dpkg_output = subprocess.run("dpkg -l | grep '^ii\\s*linux-image-[0-9]' | awk '{print $2}'", capture_output=True, text=True, shell=True)
```

Linter is complaining about `Invalid escape sequence \s`

(Déso j'ai recréer un commentaire parce que j'utilise ca comme demo pour ma formation git ;P)